lib/schemas/descriptor.rb

```diff
@@ -61,7 +62,7 @@ def retrieve
     def schema_refresher
       loop do
         refresh_schema
-        sleep(SCHEMA_REFRESH_INTERVAL)
+        sleep(SCHEMA_REFRESH_INTERVAL.to_i)
```

Comment:
hm, this shouldn't be needed, ruby automatically converts `1.hour` to 3600, but i guess there is a reason to this change?

Reply:
when value gets from yml, than it is parsed as string
if it gets default value, parsing to int will work too

Reply:
yeah, strange that i won't get parsed as an integer, but this should than fix it!